src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -155,6 +160,27 @@ export const thorchainSaversStakingOpportunitiesMetadataResolver = async ({
     }
   }
 
+  if (getConfig().REACT_APP_FEATURE_RUNEPOOL) {
+    stakingOpportunitiesById[thorchainAssetId as StakingId] = {
+      apy: undefined,
+      assetId: thorchainAssetId,
+      id: thorchainAssetId as StakingId,
+      provider: DefiProvider.ThorchainSavers,
+      // @TODO: calculate proper TVL
+      tvl: '10',
+      type: DefiType.Staking,
+      underlyingAssetId: thorchainAssetId,
+      underlyingAssetIds: [thorchainAssetId] as [AssetId],
+      rewardAssetIds: [thorchainAssetId] as [AssetId],
+      // Thorchain opportunities represent a single native asset being staked, so the ratio will always be 1
+      underlyingAssetRatiosBaseUnit: ['1'],
```

Comment:
blocker: add TODO here as well as we will need to calculate underlying asset ratios of RUNE over PoL-enabled pools

Reply:
Added in [d387f46](https://github.com/shapeshift/web/pull/7430/commits/d387f46ef66e91fa6dc499cf57b03c1153695497)